### PR TITLE
Add severe log in Action.java for legacy regkey #3135

### LINKED ISSUE #3135: Add severe log in Action.java for legacy regkey
As per title, continued from #3101.

### PR DESCRIPTION
**Pull request by jarvis57**
Fixes #3135

### CODE REVIEW

**Comment by LowWeiLin** [comment]
@thyageshm Could you do a quick check on this?

File: src/main/java/teammates/ui/controller/Action.java [hunk 1]
```diff
@@ -127,7 +127,11 @@ protected String getRegkeyFromRequest() {
         String regkey = getRequestParamValue(Const.ParamsNames.REGKEY);
         if (regkey == null) {
             //TODO: remove this branch on October 15th 2014.
-            return getRequestParamValue(Const.ParamsNames.REGKEY_LEGACY);
+            String legacyRegkey = getRequestParamValue(Const.ParamsNames.REGKEY_LEGACY);
+            if (legacyRegkey != null) {
+                log.severe("TEAMMATES accessed using old join link");
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/Action.java]: Is it possible to log who is using the legacy key?

Comment (thyageshm) [inline src/main/java/teammates/ui/controller/Action.java]: I think we can include the key in the log message as mostly we expect the user not to be loggedin. Also at this time, we do not have the student information yet

Comment (damithc) [inline src/main/java/teammates/ui/controller/Action.java]: Ok then, let's proceed with this log message.

**Comment by thyageshm** [comment]
I think it is good to go

**Pull request merged**